README.md

```diff
@@ -29,23 +29,37 @@ The copy command will take care of copying the image from internal.registry to p
         }
       }
       ```
+      To setup via method 1, you can refer to [the official kubernetes documentation](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/#log-in-to-docker-hub). 
```

Comment:
Here we should link Tekton's documentation: https://tekton.dev/docs/pipelines/auth/